cmd/chainbridge/deploy.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/urfave/cli"
+)
+
+var deployCommand = cli.Command{
+	Action:   handleDeployCmd,
+	Name:     "testdeploy",
+	Usage:    "deploys test ganache",
+	Category: "TESTING",
+	Flags:    TestFlags,
+	Description: "\tThe testdeploy command is used to deploy a ganache instance for testing\n" +
+		"\tTo add additional accounts, -testaccount\n" +
+		"\tTo only use custom accounts, -reset\n" +
+		"\tTo use a mnemonic, -mnemonic",
+}
+
+// The Private Keys of Alice, Bob, Etc
+var BaseAccounts = []string{"0x000000000000000000000000000000000000000000000000000000416c696365", "0x0000000000000000000000000000000000000000000000000000000000426f62", "0x00000000000000000000000000000000000000000000000000436861726c6965", "0x0000000000000000000000000000000000000000000000000000000044617665", "0x0000000000000000000000000000000000000000000000000000000000457665"}
```

Comment:
I think we're doing this in a few places, can these (and other instances) be pushed out to a constants file

Reply:
As David pointed out, rather than a constant file, these can be grabbed from keyring, refactoring to grab from there.